l2-frontend/src/pages/ResultsDepartment.vue

```diff
@@ -71,12 +75,13 @@
         is_lab: true,
         is_paraclinic: true,
         is_doc_refferal: false,
+        by_doc: false
       }
     },
     methods: {
-      async print() {
+      async print(flag) {
         await this.$store.dispatch(action_types.INC_LOADING)
-        const {results} = await directions_point.getDirectionsTypeDate(this, ['is_lab', 'is_paraclinic', 'is_doc_refferal', 'date']);
+        const {results} = await directions_point.getDirectionsTypeDate(this, ['is_lab', 'is_paraclinic', 'is_doc_refferal', 'date', 'by_doc']);
```

Comment:
Я уже как то описывал, как пользоваться специально созданными абстракциями апи
Можно третьим аргументом добавить какие то данные не из this (они будут влиты в итоговый объект)
```suggestion
        const {results} = await directions_point.getDirectionsTypeDate(this, ['is_lab', 'is_paraclinic', 'is_doc_refferal', 'date', 'by_doc'], {by_doc});
```